cli/nodeutil.ts

```diff
@@ -15,11 +15,10 @@ import Util = pxt.Util;
 //When running inside Electron it gets updated to the correct path
 export var targetDir: string = process.cwd();
 //When running the Electron app, this will be based on the initial value
-export var pxtCoreDir: string = path.join(targetDir, "node_modules/pxt-core")
+export var pxtCoreDir: string = path.join(targetDir, "node_modules", "pxt-core");
 
 export function setTargetDir(dir: string) {
-    targetDir = dir;
```

Comment:
Does the line not overwrite targetDir before check and if check is ok overwrites again?